arangod/Cluster/ServerState.cpp

```diff
@@ -331,6 +336,27 @@ bool ServerState::unregister() {
   return r.successful();
 }
 
+uint64_t ServerState::readRebootId(AgencyComm& comm)
+{
+  TRI_ASSERT(!_id.empty());
+  std::string rebootIdPath = currentServersRegisteredPref + _id + "/rebootId";
+  AgencyCommResult result = comm.getValues(rebootIdPath);
+
+  if (!result.successful()) {
```

Comment:
Do we get into this case if the `rebootId` does not exist?
Or will we get a `NONE` or `NULL` when accessing the path afterwards (use-case rolling upgrade)

Reply:
You will get an empty VPack as result and no error message. Keep in mind that the result is always a JSON object which contains the union values on the paths you ask for. If a path does not exist, nothing is put into the result.